src/theme.rs

```diff
@@ -27,29 +29,37 @@ pub struct Theme {
 impl Theme {
     /// Create a new theme with no colors.
     pub fn new() -> Self {
+        let placeholder = Color::Ansi256(8);
+
+        let mut focused_button = make_color(Color::Ansi256(0));
+        focused_button.set_bg(Some(Color::Ansi256(7)));
+
+        let mut blurred_button = make_color(Color::Ansi256(7));
+        blurred_button.set_bg(Some(Color::Ansi256(0)));
+
         Self {
             title: ColorSpec::new(),
             error_indicator: ColorSpec::new(),
             description: ColorSpec::new(),
             cursor: ColorSpec::new(),
-            selected_prefix: String::from(" x"),
+            selected_prefix: String::from("[•]"),
             selected_prefix_fg: ColorSpec::new(),
             selected_option: ColorSpec::new(),
-            unselected_prefix: String::from("  "),
+            unselected_prefix: String::from("[ ]"),
             unselected_prefix_fg: ColorSpec::new(),
             unselected_option: ColorSpec::new(),
             input_cursor: ColorSpec::new(),
-            input_placeholder: ColorSpec::new(),
+            input_placeholder: make_color(placeholder),
             input_prompt: ColorSpec::new(),
             help_key: ColorSpec::new(),
             help_desc: ColorSpec::new(),
             help_sep: ColorSpec::new(),
-            focused_button: ColorSpec::new(),
-            blurred_button: ColorSpec::new(),
+            focused_button,
+            blurred_button,
         }
     }
 
-    /// Create a new theme with the default colors.
+    // Create a new thee with the charm color scheme
```

Comment:
the extra slash in the comment makes it a [doc comment](https://docs.rs/demand/latest/demand/struct.Theme.html)